ewah/operators/base.py

```diff
@@ -227,8 +227,12 @@ def base_init(
         if isinstance(hash_columns, str):
             hash_columns = [hash_columns]
 
-        if include_columns and isinstance(include_columns, str):
+        if isinstance(include_columns, str):
             include_columns = [include_columns]
+            print("THIS IS THE KEY")
+            print(primary_key)
```

Comment:
Syntax (blocking): I don't think you wanted to commit this :) Please remove it

Reply:
haha ,,,!